src/telepathy/Makefile.am

```diff
@@ -46,4 +48,4 @@ telepathy_sipe_LDADD = \
 	$(DBUS_GLIB_LIBS) \
 	$(GIO_LIBS) \
 	$(GOBJECT_LIBS) \
-	$(GLIB_LIBS)
+	$(GLIB_LIBS) -lpurple
```

Comment:
Same here. Either it's purple backend and links with libpurple, or Telepathy backend and links with... well, whatever telepathy extensions need to link to.